### DISCUSSION TITLE
How to keymap to Ctrl+' (single quote)

### DISCUSSION THREAD

**Discussion by aral** [helix-editor/helix#2569] [Q&A]
Hey all,

TOML doesn’t support just any character in keys so I was wondering how you would specify a custom keymap for <kbd>Ctrl</kbd><kbd>'</kbd> (single quote).

There doesn’t seem to be a special name for the single quote (or double quote, etc.) and I’ve tried `"C-'"` without success.

On the other hand, to bind to just the single quote, `"'"` works.

**Comment by the-mikedavis** [comment]
`C-'` or `C-"` seems to not be distinguished from `'` and `"` in kitty at least:

```
$ showkey -a
Press any keys - Ctrl-D will terminate this program

' 	39 0047 0x27
" 	34 0042 0x22
' 	39 0047 0x27
' 	39 0047 0x27
" 	34 0042 0x22
```

(where the bottom ones are holding control). This may be a terminal limitation. See also https://github.com/helix-editor/helix/issues/1085
